Intersect.Editor/Forms/Editors/Events/Event Commands/Event_GraphicSelector.cs

```diff
@@ -41,20 +41,23 @@ public partial class EventGraphicSelector : UserControl
         )
         {
             InitializeComponent();
+            InitLocalization();
             mEditingGraphic = editingGraphic;
             mEventEditor = eventEditor;
             mLoading = true;
+            cmbGraphicType.SelectedIndex = (int)mEditingGraphic.Type;
+
             UpdateGraphicList();
-            if (cmbGraphic.Items.IndexOf(mEditingGraphic.Filename) > -1)
+            int count = cmbGraphic.Items.Count;
+              if (cmbGraphic.Items.Contains(mEditingGraphic.Filename))
```

Comment:
Need to fix the indentation here.. I'll test this soon and as long as it works I think it'll be fine.